### ISSUE TITLE
NTR: Renal biopsy: positive staining for PLA2R (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8784]
New term request Renal biopsy: positive staining for PLA2R (nadea3_190821160842): Phospholipase A2 receptor

**Comment by pnrobinson** [comment]
@MickeySegal
Would it be more useful to make a term for the serum antibody? See
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8815591/

**Comment by MickeySegal** [comment]
That article argues that the biopsy is needed.

**Comment by pnrobinson** [comment]
OK, after some reading, I think it is not exactly this
See: https://pubmed.ncbi.nlm.nih.gov/25997506/
The majority of cases of primary membranous nephropathy (MN) are associated with auto-antibodies against the podocyte antigen M-type phospholipase A2 receptor (PLA2R). This particular subset of MN can be diagnosed by identifying anti-PLA2R within patient sera or by detecting PLA2R antigen within glomerular immune complexes in renal biopsy tissue.

I therefore made this term:

Adding new term: 
- ID: HP:6000669
- label: Glomerrular PLA2R immune-complex deposition
- parents: [Abnormal renal glomerulus morphology (HP:0000095)](https://hpo.jax.org/app/browse/term/HP:0000095)
- pmid: PMID:25997506|PMID:21323563
- def: Accumulation of immune deposits, containing auto-antibodies against the podocyte antigen M-type phospholipase A2 receptor (PLA2R) in intraglomerular locations.
- comment: This feature is seen in the majority of cases of primary membranous nephropathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8784
- synonyms:
